x-pack/plugins/data_visualizer/public/application/common/components/fields_stats_grid/fields_stats_grid.tsx

```diff
@@ -113,6 +116,7 @@ export const FieldsStatsGrid: FC<Props> = ({ results }) => {
         pageState={dataVisualizerListState}
         updatePageState={setDataVisualizerListState}
         getItemIdToExpandedRowMap={getItemIdToExpandedRowMap}
+        width={width}
```

Comment:
Could you elaborate on why we need to use the window size here, and can't just make this 100% width to fill it's container?

Reply:
You're right, this doesn't need to depend on window width. Updated to use a size observer for the table instead here https://github.com/elastic/kibana/pull/107184/commits/d0d2289bcfa60e89ad6bd3e7a784e595a6c51842